tests/sai_qualify/conftest.py

```diff
@@ -109,9 +113,9 @@ def pytest_addoption(parser):
                      default=None, type=str,
                      help="SAI test port config file to map \
                      the relationship between lanes and interface.")
-    parser.addoption("--skip_stop_sai_test_container",
+    parser.addoption("--always_stop_sai_test_container",
```

Comment:
why did you change this?
this change happened in this PR
https://github.com/sonic-net/sonic-mgmt/commit/181d4f16204aa1d77ca64f7b1f75aa0241c9faac